src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -679,11 +647,8 @@ StaticPipelineKind str_to_pipeline(const std::string& str) {
     if (str == "STATEFUL") {
         return StaticPipelineKind::STATEFUL;
     }
-    if (str == "STATELESS") {
-        return StaticPipelineKind::STATELESS;
-    }
     OPENVINO_THROW("Unsupported \"PIPELINE\" provided: ",
-                   str, ". Please select either \"STATEFUL\" or \"STATELESS\".");
```

Comment:
Do we still need this function if we only have 1 pipeline to select from?

Reply:
Most probably we don't, but who knows, preserved it so far